README.md

```diff
@@ -73,6 +76,25 @@ This command is deprecated in favor of `ember try:one`. There are several bugs w
 
 ### Config
 
+##### versionCompatibility
+If you're using `ember-try` with an Ember addon, there is a short cut to test many Ember versions. In your `package.json` under the `ember-addon` key, add the following: 
+
+```json
+  "ember-addon": {
+    "versionCompatibility": {
+       "ember": ">1.11.0 <=2.0.0"
```

Comment:
Are there other specified patterns for this? What if we wanted to support multiple ranges? Or is that handled by YAGNI?